app/services/user.py

```diff
@@ -86,19 +86,19 @@ def __init__(self, **attrs):
         self.password = attrs.pop("password")
 
     def create(self) -> User:
-        try:
-            new_user = User(
-                email=self.email,
-                confirmed=False,
-                tier=self.get_user_tier(),
-                uuid=str(uuid.uuid4()),
-            )
-            new_user.set_password(self.password)
-            db.session.add(new_user)
-            db.session.flush()
-        except IntegrityError:
+        if User.query.filter_by(email=self.email):
```

Comment:
double check this column is indexed then 👍

Reply:
Just checked the table and it looks like it is indexed

![Screen Shot 2019-05-03 at 10 11 23 AM](https://user-images.githubusercontent.com/11577853/57154260-4eb7de80-6d8d-11e9-80f3-2b05cb6eddab.png)